### ISSUE TITLE
For fellow Mojave-updaters 

### ISSUE THREAD

**Issue by jessupjs** [susanBuck/dwa15-fall2018#71]
Just in case anyone runs into this message in Terminal after updating to macOS Mojave and trying to use git commands: "xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun".

:eyes:

A [Stack Overflow](https://apple.stackexchange.com/questions/254380/macos-mojave-invalid-active-developer-path) article offers some helpful troubleshooting.

In a nutshell, update Xcode tools: `xcode-select --install`.
